pymc3/gp/cov.py

```diff
@@ -117,6 +136,19 @@ def __array_wrap__(self, result):
             raise RuntimeError
 
 
+class Exponentiated(Covariance):
```

Comment:
Maybe move this down to be near `Add` and `Prod`? (very small nitpick, since it's more like a `Combination` covariance than a base covariance)